packages/manager/modules/vps/src/dashboard/tile/configuration/upgrade/constants.js

```diff
@@ -0,0 +1,9 @@
+export const ORDER_TRACKING_URLS = {
+  EU: 'https://www.ovh.com/manager/dedicated/#/billing/order',
+  CA: 'https://ca.ovh.com/manager/#/billing/order',
+  US: 'https://us.ovhcloud.com/manager/dedicated/#/billing/order',
```

Comment:
Maybe we should use `RedirectionService` or at least CORE_MANAGER_URLS. Especially URL for Dedicated in CA is about to move soon

Reply:
I didn't know there was this constant. I will check that. Thx

Reply:
I will wait for #2310 in order to have global const.